Libplanet.Extensions.Cocona/Commands/KeyCommand.cs

```diff
@@ -215,6 +284,21 @@ public KeyCommand()
             }
         }
 
+        [Command(Description = "Derive Public key and Address from Private key.")]
+        public void Derive(
+            [Argument(
+                "PRIVATE-KEY",
+                Description = "A raw private key to import."
+            )]
+            string key,
```

Comment:
Maybe the trailing comma should be removed.

```suggestion
            string key
```

Reply:
applied with the latter commit.